docs/changelog.rst

```diff
@@ -10,6 +10,12 @@ Unreleased
   ``QuantinuumBackend.two_qubit_gate_set`` providing the default and supported
   two-qubit gates for a device.
 * Make ``ZZPhase`` the default two-qubit gate target on all devices.
+* Add ``QuantinuumBackendCompilationConfig`` dataclass, which can be passed as
+  an optional argument when constructing a ``QuantinuumBackend``. Configuration
+  can be inspected using ``QuantinuumBackend.get_compilation_config()`` and
+  modified using the methods
+  ``QuantinuumBackend.set_compilation_config_allow_implicit_swaps()`` and
+  ``QuantinuumBackend.set_compilation_config_target_2qb_gate()``.
 * Add new methods ``QuantinuumBackend.default_compilation_pass_with_options``,
```

Comment:
The previous commit looks like it removed all three of the new `_with_options` methods, in favour of the new dataclass approach - should this bullet point be removed entirely? (Since there's not been a release with those methods in, that's presumably safe enough.)

Reply:
Oops yes, thought I'd done that.

Reply:
Done.